client/src/main/java/com/battleships/connection/commands/server/commands/ShotMessageReceived.java

```diff
@@ -14,16 +21,30 @@ public ShotMessageReceived(Object value) {
     @Override
     public void execute() {
```

Comment:
refactor it, extract if-else code to separate methods

Reply:
corrected